packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -351,16 +362,25 @@ export class ClusterReachability extends EventsScope {
 
       if (e.candidate) {
         if (e.candidate.type === CANDIDATE_TYPES.SERVER_REFLEXIVE) {
-          this.saveResult('udp', latencyInMilliseconds, e.candidate.address);
+          let serverIp = null;
+          if ('url' in e.candidate) {
```

Comment:
according to https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate url is available only in Chrome